receiver/filelogreceiver/README.md

```diff
@@ -33,6 +33,8 @@ Tails and parses logs from files.
 | `include_file_path`                 | `false`                              | Whether to add the file path as the attribute `log.file.path`.                                                                                                                                                                                                  |
 | `include_file_name_resolved`        | `false`                              | Whether to add the file name after symlinks resolution as the attribute `log.file.name_resolved`.                                                                                                                                                               |
 | `include_file_path_resolved`        | `false`                              | Whether to add the file path after symlinks resolution as the attribute `log.file.path_resolved`.                                                                                                                                                               |
+| `include_file_owner`                | `false`                              | Whether to add the file owner as the attribute `log.file.owner`  |
+| `include_file_group`                | `false`                              | Whether to add the file group as the attribute `log.file.group`  |
```

Comment:
It's probably worth calling out that we're adding the numeric ID. In fact, it may be worth changing the parameter and attribute to `include_file_owner_id` and `log.file.owner.id` so that we can differentiate from `include_file_owner_name` and `log.file.owner.name` if we ever need.

Reply:
Ah, i misread the code. I think name is better than id but we should use the naming I suggested anyways so that we have the option to add id later